admin/class-admin-asset-manager.php

```diff
@@ -318,14 +318,19 @@ protected function scripts_to_be_registered() {
 				],
 			],
 			[
-				'name' => 'recalculate',
-				'src'  => 'recalculate-' . $flat_version,
+				'name' => 'primary-category',
```

Comment:
This doesn't look right. `wp-seo-metabox-category` was removed in trunk. This looks like a merge conflict mistake.